README.md

```diff
@@ -64,8 +64,9 @@ we support!
   how-to docs and guidelines
 - [Contributing](/.github/CONTRIBUTING.md): Guidelines for making contributions
   to this repo.
-- [🏃‍♀️ Migration Guides](./docs/migration)
-  - [v9 to v10](./docs/migration/migrate-to-10.x.md)
+- 🏃‍♀️ Migration Guides
+  - [v9 to v10 (vanilla)](./packages/components/docs/migration/migrate-to-10.x.md)
+  - [v9 to v10 (React)](./packages/components/docs/migration/migrate-to-7.x.md)
```

Comment:
Should this be v6 to v7? Not sure which version we'd want to refer to here

Reply:
Yeah, our versioning here is confusing either way... In this context, adjacent to the vanilla repo it makes sense, but if a user looks at the actual version numbers it could be confusing. 

Maybe we should have it as `v9 to v10 (v6 to v7 React)` 🤷‍♂

Reply:
@tw15egan's suggestion makes sense, went ahead with it - Thanks!